src/components/pool_flux_chart.vue

```diff
@@ -66,86 +82,78 @@ export default {
 
           // pools and fluxes of water
           this.volume = data[0];
-          console.log(this.volume)
+          // console.log(this.volume)
+
+          // customize each x scale
+          this.adaptScales(this.volume, 1);
 
+          // set starting x scale
+          this.setXScale();
+          
           // draw chart
-          this.addChart(this.volume)
+          this.drawChart(this.volume, 1)
 
+          // set up radio button interaction
+          this.chartContainer.selectAll("input").on("click", this.changeXScale.bind(this));
+          
         },
-        addChart(data){
-          this.drawChart(data, {
-            x: d => d.value_km_3,
-            type: this.d3.scaleLog,
-            y_pos: 100,
-            x_min: 1, // necessary for log
-            let_class: 'pool'
-          })
+        adaptScales(data, xMin) {
+          Object.keys(this.scales).forEach(function (scaleType) {
+              this.scales[scaleType]
+                  .domain([xMin, this.d3.max(data, d => d.value_km_3)])
+                  .range([0, this.chartWidth]);
+          }, this);
```

Comment:
Interesting that you're using `this` like this. Will have to take note for the future, seems to resolve an issue we've had with self-selection with D#/Vue

Reply:
That was straight from the example -- I was please it worked!